### PR TITLE
Ytb 02 args

### LINKED ISSUE #2: YTB-2: Handle input parameters in cmd
Add supporting for running the tool from cmd with parameters. Required parameters are the following:
(-i|--input) add config file in json format

Example:
$ dglvdownloader.py -i ./config.json

### PR DESCRIPTION
**Pull request by GA10v**
добавил модуль "argparse"

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [
     {
         "tasks": [{
-            "type" : "single",
```
Comment (digal25) [inline config.json:4]: Good change!

File: config.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+            "type" : "youtube",
             "description" : "Супер Крылья",
```
Comment (digal25) [inline config.json:5]: Good changes. Type tag with **youtube**/**playlist** values looks might better.

File: ytbdownloader.py [hunk 3]
```diff
@@ -1,22 +1,40 @@
...
-def open_json():
+def get_arg():
+    parser = argparse.ArgumentParser(description='Download videos from Youtube')
+    parser.add_argument(
+        '-i', '--input', 
+        type=str, 
+        help='input name of "*.json" file (default is "/config.json") ', # default не по ТЗ, но я хотел попробывать.. сработает или нет
+
+        default='config.json' #не по ТЗ, но я хотел попробывать.. сработает или нет 
+        )
```
Comment (digal25) [inline ytbdownloader.py:14]: I like this default

File: ytbdownloader.py [hunk 4]
```diff
@@ -1,22 +1,40 @@
...
+    args = parser.parse_args()
+
+    return args
+
+def open_json(args):
 # пытается найти и открыть файл config.json 
+
+    if args.input[-5::] != '.json': #теперь можно не писать расширение *.json (не по ТЗ, но я хотел попробывать.. сработает или нет)
+        args.input += '.json'
+
```
Comment (digal25) [inline ytbdownloader.py]: I think this code is redundant. We do not need to guess file extensions. Please remove these changes related to adding .json

**Review by digal25 — CHANGES_REQUESTED** [review]

File: ytbdownloader.py (hunk 4 above)
Comment (GA10v) [reply ytbdownloader.py]: I fixed it

**Review by digal25 — APPROVED** [review]

**Pull request merged**